pom.xml

```diff
@@ -1600,6 +1600,19 @@
     </profile>
 
     <profile>
+      <id>hadoop-2.6</id>
+      <properties>
+        <hadoop.version>2.6.0</hadoop.version>
```

Comment:
@tgravescs @srowen:
org.apache.hadoop.crypto.CryptoInputStream#CryptoInputStream, org.apache.hadoop.crypto.CryptoOutputStream are two classes for encrypted shuffle which are imported since hadoop2.6.  

I used CryptoInputStream in org.apache.spark.storage.ShuffleBlockFetcherIterator and CryptoOutputStream in org.apache.spark.storage.DiskBlockObjectWriter.